cli/scan.js

```diff
@@ -0,0 +1,65 @@
+const assert = require('assert')
+const dotenv = require('dotenv')
+const { getSecurityVulnerabilitiesForMultiple } = require('../index')
+const { writeToCsv } = require('../src/csv')
+const { exitWithError } = require('../src/error')
+
+dotenv.config()
+
+const bbUsername = process.env.BITBUCKET_USERNAME
+const bbAppPassword = process.env.BITBUCKET_APP_PASSWORD
+const token = process.env.GITHUB_TOKEN
+
+const args = process.argv.splice(2)
+
+try {
+  assert(
+    process.argv.length > 2,
+    `Usage:\n  node cli/scan.js --github="org" --bitbucket="workspace" --csv="versions.url"\n\n` +
+    `  -g, --github       The Github organization to scan\n` +
```

Comment:
I wonder if it's worth adding a note about the first arg being used and the rest being ignored.